src/main/java/frc/robot/vision/rawdata/RawVisionData.java

```diff
@@ -0,0 +1,29 @@
+package frc.robot.vision.rawdata;
+
+import edu.wpi.first.math.geometry.Pose3d;
+
+public class RawVisionData {
+
+	private final Pose3d estimatedPose;
+	private final double timestamp;
+	private final boolean isDataValid;
+
+	public RawVisionData(Pose3d estimatedPose, double timestamp, boolean isDataValid) {
+		this.estimatedPose = estimatedPose;
+		this.timestamp = timestamp;
+		this.isDataValid = isDataValid;
+	}
+
+	public Pose3d getEstimatedPose() {
+		return estimatedPose;
+	}
+
+	public double getTimestamp() {
+		return timestamp;
+	}
+
+	public boolean getIsDataValid() {
+		return isDataValid;
+	}
```

Comment:
what does it mean?

Reply:
`shallDataBeFiltered` is good enough? I don't want to call it `getFilterResult` since it's not clear from the name what positive means and what negative means